website/docs/docs/build/ratio-metrics.md

```diff
@@ -55,6 +65,30 @@ metrics:
       numerator: food_orders
       denominator: orders
 ```
+</File>
+
+#### Example 2 
+This eample is a ratio metric that calculates the ratio of food orders to total orders, with a filter and alias applied to the numerator:
```

Comment:
I would just add a note at the end here to make explicit how this is different, something like:
"...with a filter and alias applied to the numerator. Note that in order to add these attributes you'll need to use an explicit key for the `name` attribute, too."